messages.js

```diff
@@ -20,10 +20,17 @@ const { deriveFeedKeyFromSeed } = require('./keys')
  * Low level API for generating messages
  */
 exports.init = function init(sbot) {
-  function optsForAdd(mfKeys, feedKeys, nonce, feedpurpose, metadata) {
-    const type = nonce ? 'metafeed/add/derived' : 'metafeed/add/existing'
+  function optsForAdd(
+    mfKeys,
+    feedKeys,
+    nonce,
+    feedpurpose,
+    metadata,
+    recps,
+    encryptionFormat
+  ) {
     const content = {
-      type,
+      type: nonce ? 'metafeed/add/derived' : 'metafeed/add/existing',
```

Comment:
@mixmix There's no action point in what I'm going to say, I just want to comment on a pattern that I find annoying. This refactor right here is merely stylistic. I wrote this part and I prefer to have the `const type`, and you just ripped that out, probably because it's not your style, without justifying it. Can I ask you to avoid doing these kind of changes that are (1) unjustified, (2) merely code style that conflicts with the original author's preference.

Reply:
This is fair critique. I have a "tidy as I go" approach but we don't agree what "tidy" means.
I'll do my best to not do this (I did consciously leave other areas alone)